src/coingecko/Coingecko.ts

```diff
@@ -275,6 +270,58 @@ export class Coingecko {
     return this.prices[platform_id][currency];
   }
 
+  protected getCachedAddressPrice(
+    contractAddress: string,
+    currency: string,
+    platform_id: string
+  ): CoinGeckoPrice | undefined {
+    const priceCache = this.getPriceCache(currency, platform_id);
+    const now: number = msToS(Date.now());
+    const tokenPrice: CoinGeckoPrice | undefined = priceCache[contractAddress];
+    if (tokenPrice === undefined || tokenPrice.timestamp + this.maxPriceAge <= now) {
+      if (this.maxPriceAge > 0) {
+        this.logger.debug({
+          at: "Coingecko#getCurrentPriceByContract",
+          message: `Cache miss on ${platform_id}/${currency} for ${contractAddress}`,
+          maxPriceAge: this.maxPriceAge,
+          tokenPrice: tokenPrice,
+        });
```

Comment:
Would we not always want to display this log?

Reply:
You mean we should also have a log when tokenPrice is undefined? We already have a log for when it is defined, I'm not sure this is needed.